net/dawn/Makefile

```diff
@@ -5,13 +5,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dawn
-PKG_SOURCE_DATE:=2022-07-24
-PKG_RELEASE:=3
+PKG_SOURCE_DATE:=2023-05-14
+PKG_RELEASE:=4
```

Comment:
Please set PKG_RELEASE to 1 this is a new upstream version. PKG_RELEASE should only increment if we do not change the upstream version. And if we have a new upstream version the we reset the PKG_RELEASE to 1

Reply:
Fixed and force-pushed for better commit message.